xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -1949,71 +1943,98 @@ void COutput::InitCycle()
       if (method == VS_INTERLACEMETHOD_DEINTERLACE ||
           method == VS_INTERLACEMETHOD_RENDER_BOB)
       {
+        CLog::Log(LOGINFO, LOGVIDEO, "VAAPI output: Initializing ffmpeg postproc");
```

Comment:
this should not be LOGINFO, make it component logging for video as the other logs here

Reply:
It is LOGVIDEO